### PR TITLE
update state drop

### LINKED ISSUE #863: Create a state dropdown
# Problem

#849 added a basic state dropdown that needs to be populated with actual data.

https://github.com/code-corps/code-corps-ember/pull/849/files#diff-fadc9970a9b163a9321d697cbdd42755R63

### PR DESCRIPTION
**Pull request by rileytaylor**
# What's in this PR?

This updates the state dropdown to include a list of states and their corresponding postal codes.

## References
Fixes #863 

Progress on: #853

### CODE REVIEW

File: app/components/select/state-select.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
+
+const {
+  Component
+} = Ember;
+
+export default Component.extend({
+  classNames: ['select-state'],
+  stateOptions: [
+    { name: 'Alabama', id: 'AL' },
```
Comment (JoshSmith) [inline app/components/select/state-select.js]: I would note this article link in here for documentation: https://support.stripe.com/questions/can-i-use-stripe-if-my-business-is-based-in-an-outlying-territory-of-a-supported

File: tests/integration/components/select/state-select-test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+let page = PageObject.create(selectState);
+
+moduleForComponent('select/state-select', 'Integration | Component | select/state select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.setContext(this);
```
Comment (JoshSmith) [inline tests/integration/components/select/state-select-test.js]: This should be `page.removeContext();`. We may have this wrong in some other tests. Like country, it looks like. Can fix in this PR.

File: tests/integration/components/select/state-select-test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+let page = PageObject.create(selectState);
+
+moduleForComponent('select/state-select', 'Integration | Component | select/state select', {
+  integration: true,
+  beforeEach() {
+    page.setContext(this);
+  },
+  afterEach() {
+    page.removeContext(this);
```
Comment (JoshSmith) [inline tests/integration/components/select/state-select-test.js]: `page.removeContext()`

Comment (rileytaylor) [inline tests/integration/components/select/state-select-test.js]: oh duh.

**Comment by JoshSmith** [comment]
@rileytaylor LGTM. Merge when passing.

**Pull request merged**